Automattic/wp-calypso
# Add/user licensing banner

#### Changes proposed in this Pull Request

Add a banner to the checkout flow and pricing page that informs a user when they have unused licenses that can be activated.

#### Testing instructions



Ensure that the banner adjusts and is displayed correctly for the mobile experience with both the Jetpack Connect flow and the plans page flow. 

- Download the PR
- Start both Calypso and Jetpack Cloud ```yarn start``` and ```yarn start-jetpack-cloud-p``` in order
- Ensure that you have a purchase for a plan that has a license that has not yet been activated or associated with a site. 

Plans Page Flow
- Create a new JN site and connect the site. 
- Once the site is connected go to the plans page. Once on the plans page update the url from https://cloud.jetpack.com to http://jetpack.cloud.localhost:3001 
-  Ensure that the banner is present and adjust correctly for mobile. 

Jetpack Connect Flow
- Attempt to connect a disconnected site. Once on the page to authorize the connection update the url from wordpress.com to http://calypso.localhost:3000/
- Authorize the connection and ensure that you are taken to the pricing page and that the banner indicating an available license can be activated is present. Ensure that the banner adjust correctly for mobile




Related to #
